boards/shields/frdm_stbc_agm01/frdm_stbc_agm01.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_i2c {
+    fxos8700@1e {
+		compatible = "nxp,fxos8700";
+		reg = <0x1e>;
+		label = "AGM01_FXOS8700";
+		int1-gpios = <&arduino_header 8 GPIO_ACTIVE_LOW>;
+		int2-gpios = <&arduino_header 10 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&arduino_header 3 GPIO_ACTIVE_LOW>;
```

Comment:
The default jumper placement on the shield disconnects reset-gpio and utilizes mcu reset, instead.

This reset-gpio is also connected to the fxas21002, so it's probably best to not use it here, anyways.